src/components/Slide/Slide.styles.scss

```diff
@@ -0,0 +1,6 @@
+:host {
+    display: none;
+}
+:host(.active) {
```

Comment:
Q: Since we're using sass, can we use `.active` inside `:host`, or is it not the same thing?

Reply:
Nope, this wont work